public/locales/auset-te.json

```diff
@@ -0,0 +1,426 @@
+      "setupTitle": "మార్క్వైస్ డి క్యాట్ ని సెటప్ చేయండి",
+      "setup": "<i>1.</i> 3 హోమ్‌ల్యాండ్ క్లియరింగ్‌లను ఎంచుకోండి, ప్రతి ఒక్కటి ఒకదానికొకటి ప్రక్కనే ఉంటుంది.<br/><i>2.</i> మీ ప్రతి స్వదేశంలో 2 <strong>యోధులను</strong> ఉంచండి. ఒకదానికొకటి క్లియరింగ్‌లో 1 <strong>యోధుడిని</strong> ఉంచండి.<br/><i>3.</i> <strong>కీప్ టోకెన్</strong>ని మీ స్వదేశంలో ఒకదానిలో ఉంచండి, శత్రువుకు ఆనుకొని ఉండకూడదు వీలైతే మాతృభూమి. మ్యాప్‌లో 1 <strong>సామిల్</strong>, <strong>వర్క్‌షాప్</strong> మరియు <strong>రిక్రూటర్</strong>ని ఉంచండి, ఒక్కొక్కటి మీ స్వంత స్వదేశంలో.<br/><i>4. </i> ఎడమవైపు ఖాళీలు మినహా, మీ <strong>భవనాలు</strong>తో మీ భవనాల ట్రాక్‌ని పూరించండి."
+    },
+    "riverfolk": {
+      "name": "రివర్‌ఫోక్ కంపెనీ",
+      "summaryTitle": "రివర్‌ఫోక్ కంపెనీ గా ఆడటం",
+      "summary": "<p>రివర్‌ఫోక్ కంపెనీగా, మీరు గ్రేట్ వుడ్‌ల్యాండ్ గుండా నదుల్లో తిరుగుతూ, చెల్లించగల ఏ వర్గానికైనా మీ సేవలను అందిస్తారు. క్లియరింగ్‌ల అంతటా <strong>ట్రేడ్ పోస్ట్‌లను</strong> స్థాపించడం ద్వారా మీరు విజయ పాయింట్‌లను స్కోర్ చేస్తారు.</p><p>పాయింట్‌లను స్కోర్ చేయడానికి ట్రేడ్ పోస్ట్‌ల నిర్మాణం ఆచరణీయమైన మార్గం అయినప్పటికీ, సంపద యొక్క ముడి సేకరణ కూడా. ప్రతి పక్షుల పాట(బర్డ్ సాంగ్), మీరు గత రౌండ్‌లో ఎన్ని <strong>ఫండ్‌లు</strong> ఆదా చేసారు మరియు సంపాదించారు అనే దాని ఆధారంగా మీరు పాయింట్‌లను స్కోర్ చేస్తారు. అయినప్పటికీ, మీరు డివిడెండ్‌లు మరియు వృద్ధి మధ్య సమతుల్యతను సాధించడం ద్వారా మీ వ్యాపార నెట్‌వర్క్‌ను విస్తరించడానికి మరియు రక్షించుకోవడానికి మీ నిధులను కూడా కట్టుబడి మరియు ఖర్చు చేయాలి.</p><p>ఇతర వర్గాలు మీ <strong>సేవలను కొనుగోలు చేయడం వలన మీరు నిధులను సంపాదిస్తారు. </strong>, మీ రివర్ బోట్‌లు లేదా కిరాయి యోధుల ఉపయోగం లేదా మీ పబ్లిక్ చేతిలో ఉన్న కార్డ్‌లలో ఒకదానిని ఉపయోగించినప్పటికీ, ఇది <strong>అమ్మకానికి</strong> ఉంది. మీరు మీ సేవల ధరలను ప్రతి మలుపులో సెట్ చేయవచ్చు, కాబట్టి ఇతర వర్గాలకు ఏయే సేవలు అవసరమో అంచనా వేయడానికి ప్రయత్నించండి.</p><p>వర్తక పోస్ట్‌లను నిర్మించడం వలన మీ క్లయింట్‌లను విస్తరించడంలో మరియు మరిన్ని సేవలను విక్రయించడంలో మీకు సహాయపడుతుంది, ఎందుకంటే ఒక వర్గం మరిన్ని సేవలను కొనుగోలు చేయగలదు. మీరు ట్రేడ్ పోస్ట్‌లను కలిగి ఉన్న మరిన్ని క్లియరింగ్‌లకు విస్తరిస్తున్నప్పుడు మీ నుండి. అయితే జాగ్రత్తగా ఉండండి, మీ ట్రేడ్ పోస్ట్‌లు నాశనం అయినప్పుడు వాటిని పునర్నిర్మించలేము! మీరు అనేక క్లియరింగ్‌లను పాలించకపోయినప్పటికీ, మీరు <strong>ఈతగాళ్ల</strong> వర్గంగా ఉన్నందున మీరు ఎల్లప్పుడూ నదీ మార్గాల్లో వెళ్లవచ్చు.</p>",
+      "setupTitle": "రివర్‌ఫోక్ కంపెనీని ఏర్పాటు చేయండి",
+      "setup": "<i>1.</i> నది వెంబడి ఏవైనా క్లియరింగ్‌లలో 4 <strong>యోధులను</strong> ఉంచండి.<br/><i>2.</i> 3 <strong>యోధులను</strong> ఉంచండి. మీ చెల్లింపుల పెట్టె.<br/><i>3.</i> సరిపోలే <strong>ట్రేడ్ పోస్ట్‌లు</strong>తో మీ ట్రేడ్ పోస్ట్‌ల ట్రాక్‌లను పూరించండి.<br/><i>4.</i> మీ 3ని ఉంచండి. మీ సేవల ట్రాక్‌లో <strong>సేవా గుర్తులు</strong>, ప్రతి సేవకు ధరను సెట్ చేస్తుంది."
+    },
+    "vagabond": {
+      "name": "సంచారి(వాగాబాండ్)",
```

Comment:
I think the phonetic translation is a nice touch, however it creates odd looking nested brackets when included in the faction select:
![image](https://user-images.githubusercontent.com/39972488/176201274-fda446ef-a08f-47b1-a4c0-872c09eedd1b.png)

and sometimes it even causes an overflow from being too long:
![image](https://user-images.githubusercontent.com/39972488/176201425-f1310ffd-7a11-4b0b-905e-b3ff7a87d9a8.png)

I would suggest just using the faction name from the official Leder games translation, or if that doesn't exist use just the literal translation without the phonetic one.